<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1920</width>
    <height>1080</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgba(0, 0, 0, 255);</string>
  </property>
  <widget class="QPushButton" name="close_">
   <property name="geometry">
    <rect>
     <x>1870</x>
     <y>10</y>
     <width>40</width>
     <height>40</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius: 20px;
background: none;
background-color: rgb(255, 255, 255);
border-style: none;</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>../icons/remove-button.png</normaloff>../icons/remove-button.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>36</width>
     <height>36</height>
    </size>
   </property>
  </widget>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>1850</width>
     <height>40</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background: none;
background-color: rgba(0, 0, 0, 0);</string>
   </property>
  </widget>
  <widget class="QScrollArea" name="cryptosScroll">
   <property name="geometry">
    <rect>
     <x>940</x>
     <y>10</y>
     <width>920</width>
     <height>520</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background: none;
background-color: rgba(0, 0, 0, 0);</string>
   </property>
   <property name="widgetResizable">
    <bool>true</bool>
   </property>
   <widget class="QWidget" name="scrollAreaWidgetContents_4">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>918</width>
      <height>518</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QScrollArea" name="forexScroll">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>9</y>
     <width>920</width>
     <height>520</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background: none;
background-color: rgba(40, 40, 40, 10);</string>
   </property>
   <property name="widgetResizable">
    <bool>true</bool>
   </property>
   <widget class="QWidget" name="scrollAreaWidgetContents_5">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>918</width>
      <height>518</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QScrollArea" name="indicesScroll">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>540</y>
     <width>920</width>
     <height>520</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background: none;
background-color: rgba(0, 0, 0, 0);</string>
   </property>
   <property name="widgetResizable">
    <bool>true</bool>
   </property>
   <widget class="QWidget" name="scrollAreaWidgetContents_6">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>918</width>
      <height>518</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QScrollArea" name="commoditiesScroll">
   <property name="geometry">
    <rect>
     <x>940</x>
     <y>540</y>
     <width>920</width>
     <height>520</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background: none;
background-color: rgba(0, 0, 0, 0);</string>
   </property>
   <property name="widgetResizable">
    <bool>true</bool>
   </property>
   <widget class="QWidget" name="scrollAreaWidgetContents_7">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>918</width>
      <height>518</height>
     </rect>
    </property>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="../rsrc/images/backgrounds.qrc"/>
 </resources>
 <connections/>
</ui>
